From 2886528703fa04bcc786bc80f11279a78fabc452 Mon Sep 17 00:00:00 2001
From: h3ll <h3ll@sideros.org>
Date: Sat, 15 Mar 2025 02:35:57 +0200
Subject: [PATCH] removed unused code or something

---
 src/render/vulkan.zig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/render/vulkan.zig b/src/render/vulkan.zig
index e9993a5..0730c8c 100644
--- a/src/render/vulkan.zig
+++ b/src/render/vulkan.zig
@@ -502,7 +502,7 @@ pub fn GraphicsPipeline(comptime n: usize) type {
     };
 }
 
-//pub const Shader = struct {
+// removed line was here :(
 //    handle: c.VkShaderModule,
 //
 //    pub fn create(comptime name: []const u8, device: Device) !Shader {
-- 
2.48.1

